src/assets/source/sass/components/_about.scss

```diff
@@ -0,0 +1,36 @@
+#about {
+  .card {
+    max-width: 1500px;
+    opacity: 0.9;
+    width: 90%;
+    margin: 7% 10% 0;
+    padding-bottom: 13%;
+
+    h3 {
+      letter-spacing: 3px;
+      color: rgba(61, 61, 61, 0.87);
```

Comment:
You use this value in many places - please add it to a variable and use the variable instead hardcoding it.